tests/compiler/progName.lean.expected.out

```diff
@@ -0,0 +1 @@
+./progName.lean.out: [first, second]
```

Comment:
Any tips for writing this test? Unfortunately the windows test fails as it outputs
```
D:\a\lean4\lean4\tests\compiler\progName.lean.out: [first, second]
```
which seems quite fragile.